### PR TITLE
Issue/1649 & Issue/1650: remove consoles.min.js and json2.min.js

### PR DESCRIPTION
**Pull request by akshay-bhavsar**
#1649 #1650 
remove json2.js 
remove consoles.js
and changes in scriptLoader.js

### CODE REVIEW

**Comment by oliverfoster** [comment]
you could probably do consoles.js as well in here? save having conflicting prs?

**Comment by oliverfoster** [comment]
and jquery v1?

**Comment by akshay-bhavsar** [comment]
@oliverfoster sure

**Comment by sarveshwar-gavhane** [comment]
jquery v1 already done in https://github.com/adaptlearning/adapt_framework/pull/1635

File: src/core/js/scriptLoader.js [hunk 1]
```diff
@@ -118,20 +118,12 @@
         Modernizr.load('adapt/js/adapt.min.js');
     }
 
-    //1. Load foundation libraries, json2, consoles, requirejs
```
Comment (sarveshwar-gavhane) [inline src/core/js/scriptLoader.js]: please remove consoles

**Pull request merged**